src/api/app/controllers/webui/package_controller.rb

```diff
@@ -116,9 +120,7 @@ def dependency
       next if project_repositories.include?(params[repo_key])
       flash[:error] = "Repository '#{params[repo_key]}' is invalid."
       redirect_back(fallback_location: project_show_path(project: @project.name))
-      # rubocop:disable Lint/NonLocalExitFromIterator
```

Comment:
I would prefer to keep this here, instead of disabling the cop for the whole file

Reply:
hahaha that's tricky @bgeuken if I had inline somebody would ask me to remove it from inline, and put in the todo. But I prefer inline too :)

Reply:
I agree with you, I just would prefer to disable and enable it as ```prologue``` and ```epilogue``` to the function (before and after) just to avoid miss them if they are deeply nested in a method before some scary code..

Reply:
> if I had inline somebody would ask me to remove it from inline

Nobody did. I guess this is settled :cowboy_hat_face:

Reply:
@bgeuken I just fear that the PR to enable ```pundit verify_authorized``` isn't the right place for that.. if we are going to get 10000 comments here, I prefer to refactor the method :satisfied: :satisfied: